History.md

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
We probably don't necessarily need a History.md file, since we can view the history right in Github, but it does remind me that I should probably do release notes when we're ready to submit v1.1.

Reply:
Removed History.md. I think I added it somehow by mistake